Tests/PrayKitTests/Models/PrayerTimerTests.swift

```diff
@@ -605,7 +605,7 @@ extension PrayerTimerTests {
 private extension PrayerTimerTests {
     func assertCalculations(for prayerTimer: PrayerTimer, minutes: Int, lastAdhanTime: Date? = nil) {
         XCTAssertEqual(prayerTimer.timeRemaining, Double(minutes) * 60)
-        XCTAssertEqual(prayerTimer.timeRange.formatted(Date.ComponentsFormatStyle(style: .condensedAbbreviated, fields: [.minute])), "\(abs(minutes))min")
+        XCTAssertEqual(prayerTimer.timeRange.formatted(Date.ComponentsFormatStyle(style: .condensedAbbreviated, fields: [.minute])).isDigit, "\(abs(minutes))")
```

Comment:
Instead of creating a custom extension `.isDigit`, you can remove non-digits using native `CharacterSet`:

```swift
..., fields: [.minute])).components(separatedBy: .decimalDigits.inverted).joined(), "\(abs(minutes))")
```